test/synth/__snapshots__/synth-stdout.test.ts.snap

```diff
@@ -16104,6 +16104,142 @@ metadata:
 }
 `;
 
+exports[`Helm synthesis --chart-name is used when specified with all inputs from cli and no config file is present 1`] = `
+Object {
+  "Chart.yaml": "apiVersion: \\"v2\\"
+name: \\"cdk8s-synth-test\\"
```

Comment:
I believe this isn't the expected value here - it should be `custom-chart-name`? 

Its because you need to add `chartName` here:

https://github.com/cdk8s-team/cdk8s-cli/blob/9f54e63104543548b4d15b3e60fc8ff9a49f9cc8/test/synth/synth-stdout.test.ts#L1088

Otherwise, the cli input tests don't actually pass it to the synth handler.